lib/cookieContentHandler.js

```diff
@@ -46,6 +47,11 @@ function CookieContentHandler(maxAmount, maxAge, cookieEncryptionKeys, domain) {
   if (!cookieEncryptionKeys || !Array.isArray(cookieEncryptionKeys) || cookieEncryptionKeys.length === 0)
     throw new Error('CookieContentHandler: cookieEncryptionKeys must be a non-emptry array');
 
+  if (typeof cookieSameSite !== 'boolean') {
+    throw new Error('CookieContentHandler: cookieSameSite must be a boolean');
+  }
```

Comment:
Instead of throwing, you can just cast whatever `cookieSameSite` is to its boolean equivalent: 

```js
this.cookieSameSite = !!cookieSameSite;
```

Reply:
yeah I could but my concern is that they pass in a string 'none' 'lax' 'strict'

Reply:
Maybe rename this to `enableSameSitCookie` to help make it clear that it should be a boolean?

Reply:
It actually looks like you are already casting to a boolean below, so this condition will never evaluate to false.

Reply:
so two things:
 1. for the name of the value, I chose the current name based on the other options for consistency  such as `cookieDomain` which is a similar option for setting cookie.  Also i left this sort of vague on purpose.  The express cookie for `sameSite` will take a `true` `false` OR `'none'` `'lax'` or `'strict'` .  Even though my current option only supports sameSite none  I wanted to leave the option in the future to open it up to one of these other values if requested by customers

2. Even though this is cast a level up,  I did want to make sure cookiecontenthandler enforced its contract in the scenario future consumers are not casting